examples/wallet/import-secp256k1-bip44-y.in.static

```diff
@@ -0,0 +1 @@
+oasis wallet import emma --algorithm secp256k1-bip44 --number 0 --secret "man ankle mystery favorite tone number ice west spare marriage control lucky life together neither" -y
```

Comment:
Since we're using the same mnemonic and other parameters, let's also use the same account name as above to avoid confusion.

```suggestion
oasis wallet import eugene --algorithm secp256k1-bip44 --number 0 --secret "man ankle mystery favorite tone number ice west spare marriage control lucky life together neither" -y
```